services/data_plane_manager/src/test/java/com/futurewei/alcor/dataplane/utils/DataPlaneManagerUtil.java

```diff
@@ -79,21 +120,35 @@ public NetworkConfiguration autoGenerateUTsInput(int operationType,
             InternalRouterInfo routerInfo = new InternalRouterInfo();
             InternalRouterConfiguration internalRouterConfiguration = new InternalRouterConfiguration();
             List<InternalSubnetRoutingTable> subnet_routing_tables = new ArrayList<>();
+            for (int i = 0; i < subnetNum; i ++) {
```

Comment:
@kevin-zhonghao As this change is a pretty big change, could you please generate a few input and output using the UT framework? Let us review them together tomorrow.

Reply:
NP